.github/workflows/helm-docs.yaml

```diff
@@ -0,0 +1,45 @@
+name: Create helm-docs
+
+on:
+  pull_request:
+  merge_group:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  helm-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
+        with:
+          app_id: ${{ secrets.PROJECT_ACTIONS_MANAGEMENT_APP_ID }}
+          private_key: ${{ secrets.PROJECT_ACTIONS_MANAGEMENT_APP_PEM }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ steps.generate_token.outputs.token }}
+
+      - name: Run helm-docs from go
+        working-directory: charts/studio/
+        run: |
+          go run github.com/norwoodj/helm-docs/...@v1.11.0
+
+      - name: Run helm-docs
+        working-directory: charts/studio/
+        run: |
+          docker run --pull always --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Helm-Docs update
+          file_pattern: charts/studio/README.md
```

Comment:
We probably don't need this, nor the token, because `cml pr` below does everything.

Reply:
```suggestion
```

Reply:
My bad: reviewing two files at once 🤦🏼‍♂️